### ISSUE TITLE
Identifiers are not case-sensitive while variable parameter names are

### ISSUE THREAD

**Issue by stratosn** [sdmx-twg/sdmx-vtl#14]
from|No|doc|UM|RM|page/line|excel ref
----|----|----|----|----|-----|----
PL-CSO|5|UM|46||1575, 3582|50

Identifiers are not case-sensitive while variable parameter names are. PP 46-103
Update ‘ Conventions for the grammar of the language’ section, line 3582: Parameter names are not case-sensitive.

**Comment by bellomarini** [comment]
I agree with the comment PL-CSO-5 in the sense that there are some incoherent conventions.

The suggestion is:

* the names of the components are case insensitive;
* the names of the datasets are case insensitive;
* the names of the variable parameters are case insensitive;

**Comment by amattioc** [comment]
fixed in 2.0 manual
